### PR TITLE
Add support for pytest-xdist 2.1.0

### PR DESCRIPTION
**Pull request by jsfehler**

### CODE REVIEW

**Comment by bubenkoff** [comment]
same comments about the changelog - please follow the flow

**Review by bubenkoff — CHANGES_REQUESTED** [review]
same comments about the changelog - please follow the flow

**Comment by jsfehler** [comment]
@bubenkoff Changes made.

File: setup.py [hunk 1]
```diff
@@ -31,6 +31,9 @@
         'pytest>=3.0.0',
         'urllib3',
     ],
+    extras_require={
+        'xdist': ['pytest-xdist>=2.1.0'],
```
Comment (bubenkoff) [inline setup.py:35]: please reflect this in the test requirements then, as apparently it was installing the older version or why is it not catching the incompatibility ATM?

**Review by bubenkoff — CHANGES_REQUESTED** [review]

**Pull request closed without merging**